ui/src/api-logic/mappers/users.ts

```diff
@@ -0,0 +1,38 @@
+import { UserData } from 'src/@types'
+import { USER_TYPE } from 'src/utils/constants'
+import { UsersQuery } from '../graphql/generated/schema'
+
+/**
+ * Method which iterate each user and map it to UserData to be shown in table
+ * @param data User list that comes from data base in UserData format
+ * @returns User list from data base mapped to RepoImportData list
+ */
+const mapToUserData = (data: UsersQuery | undefined): Array<UserData> => {
+  const mappedData: Array<UserData> = []
+
+  data?.userMgmtPgUsers?.nodes.forEach((usr) => {
+    if (usr.memberof && usr.memberof.length > 0) {
+      // Consolidated User info
+      const userInfo: UserData = {
+        name: usr.rolname || '',
+        role: getRol(usr.memberof)
+      }
+      mappedData.push(userInfo)
+    }
+  })
+
+  return mappedData
+}
+
+const getRol = (rol: Array<string | null>) => {
+  switch (rol[0]) {
```

Comment:
@german-mergestat I think we need to check all roles in the `memberof` here - it's possible the user may have other roles and that the ones we're looking for are not the first (or only). We should check for the presence of the role in the whole list.

Reply:
Done [9f3646c](https://github.com/mergestat/mergestat/pull/556/commits/9f3646cfb0f051f1cd97afd0b932de6f376f60ef)